doc/source/release-notes/Percona-Server-5.7.33-36.rst

```diff
@@ -0,0 +1,34 @@
+.. _PS-5.7.33-36:
+
+================================================================================
+*Percona Server for MySQL* 5.7.33-36
+================================================================================
+
+:Date: March 1, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.33-36
+includes all the features and bug fixes available in
+`MySQL 5.7.33 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-33.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-5364`: Update Hashicorp Vault plugin to use v2 API (Thanks to user aprokofyev for reporting this issue)
```

Comment:
To be honest PS-5364 title is a bit misleading, currently Hahicorp Vault does not have v2 API. The fix is about supporting `KV Secrets Engine Version 2` while Server API being used is still ` v1`.

Reply:
So the wording  should be "Update Hashicorp Vault plugin to support v2 API"?

Reply:
@nastena1606 I've just updated https://jira.percona.com/browse/PS-5364 title
The correct wording should be `Update keyring_vault plugin to support KV Secrets Engine Version 2 (kv-v2)`

Reply:
Fixed